src/ripple/app/misc/HashRouter.h

```diff
@@ -172,7 +162,7 @@ class HashRouter
     // Stores all expiration times and the hashes indexed for them
     std::map< int, std::list<uint256> > mSuppressionTimes;
 
-    int mHoldTime;
+    int const mHoldTime;
```

Comment:
If you make a data member const you should explicitly `delete` the copy assignment operator